.github/workflows/arm-runner.yml

```diff
@@ -88,22 +90,22 @@ jobs:
           cpu: arm1176
           cpu_info: cpuinfo/raspberrypi_zero_w
           base_image: dietpi:rpi_armv6_bullseye
-        - target: zero2_raspios
-          cpu: cortex-a7
-          cpu_info: cpuinfo/raspberrypi_zero2_w
-          base_image: raspios_lite:latest
-        - target: zero2_dietpi
-          cpu: cortex-a7
-          cpu_info: cpuinfo/raspberrypi_zero2_w
-          base_image: dietpi:rpi_armv7_bullseye
-#        - target: zero2_64_raspios
-#          cpu: cortex-a53
-#          cpu_info: cpuinfo/raspberrypi_zero2_w_arm64
-#          base_image: raspios_lite_arm64:latest
-#        - target: zero2_64_dietpi
-#          cpu: cortex-a53
-#          cpu_info: cpuinfo/raspberrypi_zero2_w_arm64
-#          base_image: dietpi:rpi_armv8_bullseye
+#        - target: zero2_raspios
```

Comment:
We don't want to keep so many commented lines in source control. Either we delete these lines or we keep them.

Reply:
Feel free to suppress them if you wish...